<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TESTING-029" name="Demo.Ex27" tests="16" failures="1" timestamp="19 Aug 2019 05:46:32 GMT" time="751.527" errors="0">
  <testcase name="BrowserLaunch" time="1.135" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.254" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.660" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.150" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="3.870" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.541" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.547" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.544" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.180" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="2.299" classname="Demo.Ex27"/>
  <testcase name="BrowserLaunch" time="1.013" classname="Demo.Ex27"/>
  <testcase name="LoginApplication" time="323.763" classname="Demo.Ex27"/>
  <testcase name="TC_QCstage" time="77.778" classname="Demo.Ex27"/>
  <testcase name="TC_CRA1st" time="128.343" classname="Demo.Ex27"/>
  <testcase name="TC_L3" time="67.186" classname="Demo.Ex27"/>
  <testcase name="TC_RSM" time="136.264" classname="Demo.Ex27">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at Demo.Ex27.TC_RSM(Ex27.java:447)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_RSM -->
</testsuite> <!-- Demo.Ex27 -->
